mmv1/products/pubsub/Topic.yaml

```diff
@@ -166,6 +170,9 @@ properties:
         type: NestedObject
         description: |
           Settings for ingestion from Amazon Kinesis Data Streams.
+        exactly_one_of:
```

Comment:
This is potentially a breaking change if `aws_kinesis` was previously optional. Could we use `conflicts` instead? That will make the behavior "at most one of" instead of "exactly one of".

Reply:
Changed it to `conflicts`